src/js/apps/patients/worklist/filters_app.js

```diff
@@ -84,6 +89,27 @@ export default App.extend({
 
     this.showChildView('role', roleFilter);
   },
+  showOwnerFilterView() {
+    const clinician = Radio.request('entities', 'clinicians:model', this.getState('clinicianId'));
+    const role = Radio.request('entities', 'roles:model', this.getState('roleId'));
+
+    const ownerFilter = new OwnerDroplist({
+      owner: clinician.isValid() ? clinician : role,
```

Comment:
maybe just `clinicianId ?`. `isValid` is just checking to see if the model can be saved to the server.  Really you could make a `_getOwner` perhaps that `if (clinicianId) return Radio.request.. ; return Radio.request(... roles`

Reply:
oh yeah...I did this before I thought about unsetting the clinicianId/roleId on state. That makes sense